PowerShell/JumpCloud Module/Public/Commands/Invoke-JCDeployment.ps1

```diff
@@ -149,7 +149,7 @@ The CommandID will be the 24 character string populated for the _id field.')]
         $ExistingCommand = Get-JCSDKCommand -Id $CommandID
 
         # set command w/ origional type and new trigger
-        $Command = Set-JCSDKCommand -ID $CommandID -launchType trigger -trigger $trigger -Command $ExistingCommand.Command1 -CommandType $ExistingCommand.CommandType -Shell $ExistingCommand.Shell
+        $Command = Set-JCSDKCommand -ID $CommandID -launchType trigger -trigger $trigger -Command $ExistingCommand.Command1 -CommandType $ExistingCommand.CommandType -Shell $ExistingCommand.Shell -Name $ExistingCommand.Name
```

Comment:
The Invoke-JCDeployment tests had been skipped in the main branch. Function updated to include a required field in the API "name"